src/garage/_functions.py

```diff
@@ -43,7 +41,6 @@ def log_performance(itr, batch, discount, prefix='Evaluation'):
         tabular.record('MaxReturn', np.max(undiscounted_returns))
         tabular.record('MinReturn', np.min(undiscounted_returns))
         tabular.record('CompletionRate', np.mean(completion))
-        if np.mean(success) > 0:
-            tabular.record('SuccessRate', np.mean(success))
+        tabular.record('SuccessRate', np.mean(success))
```

Comment:
This will log `SuccessRate` even when there is no `success` field in `env_infos`. I suggest using a flag to control logging it.

Reply:
The way that KR has written the array `success` , so that it is empty whenever there is no field called "success" in env_infos. I think this is fine because therefore, when env_infos is logged, then the success rate will be 0. 

Also consider this, what if environment only includes success in the env_infos dictionary after a certain number of timesteps taken. On our end, we wouldn't be able to log this if `success rate` wasn't a metric, because dowel isn't set up to add keys to the logger after the first training iteration.

Reply:
I think Keren is right that this will cause spam as-is. I looked through all the Meta-World environments, and they all seem to consistently output this key. Since we only need one environment to output the key in the first batch, I think this change is fine. However, I'm wondering if we should make log warnings (such as new keys) into errors in the runner in the future.